(GH-2032) Document using Puppet Forge modules in YAML plans (#2053)
puppetlabs/bolt: documentation/applying_manifest_blocks.md

Ran on 2 targets in 8.42 sec
```

## Applying Puppet code from Puppet Forge modules in a YAML plan

In addition to the `bolt apply` command, you can apply Puppet code from a

> module downloaded from the Puppet Forge in a YAML plan. Puppet code can be
> applied using the `resources` YAML plan step.

Reviewer: ```suggestion
module downloaded from the Puppet Forge in a YAML plan. To apply 
Puppet code in a YAML plan, use the `resources` YAML plan step.
```
